http/cors.go

```diff
@@ -0,0 +1,90 @@
+package http
+
+import (
+	"errors"
+	"strings"
+
+	cs "github.com/cloudtrust/common-service/v2"
+	"github.com/rs/cors"
+)
+
+// Configuration interface
+type Configuration interface {
+	GetBool(key string) bool
+	GetStringSlice(key string) []string
+}
+
+// CorsConfiguration struct
+type CorsConfiguration struct {
+	AllowedOrigins   []string `mapstructure:"allowed-origins"`
+	AllowedMethods   []string `mapstructure:"allowed-methods"`
+	AllowCredentials bool     `mapstructure:"allow-credential"`
+	AllowedHeaders   []string `mapstructure:"allowed-headers"`
+	ExposedHeaders   []string `mapstructure:"exposed-headers"`
+	Debug            bool     `mapstructure:"debug"`
+}
```

Comment:
I don't think those comments are meaningful. If you want to keep documentation, it could be nice to add more information about what are those fields